docs/design/README.md

```diff
@@ -256,6 +255,36 @@ etc.
 These metadata files will give twoliter the information it needs to resolve, pull, and aggregate all dependency kits and the correct SDK needed for a build.
 This dependency resolution will be modeled in Cargo and kept in a `Cargo.lock` file (see [Twoliter Update](#twoliter-update)).
 
+### Kit Aggregation when Building Variants
```

Comment:
Change this. It will require multiple Dockerfiles because a kit can depend on an external kit during it's build, and a local kit can depend on another local kit.

Instead, copy RPMs out to the build directory in a kit-namespaced yum repo structure.